config/service-brokers/sqs/prod-lon-config.json

```diff
@@ -4,6 +4,7 @@
       "gds-document-checking-service",
       "gds-tech-ops",
       "gds-techarchs",
-      "nicts-probate"
+      "nicts-probate",
+      "ccs-conclave-sssos"
```

Comment:
Can we keep it alphabetised?

Reply:
Andy, you're a more sensible man than I.

Fixed.